go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/celestiaorg/celestia-app/v3
 
 go 1.23.6
 
+toolchain go1.24.1
```

Comment:
Should we bump the version in the go directive? And in the README?

Reply:
yeah I think so. I'll export this into an issue let's bump go version and then rebase these prs?

Reply:
https://github.com/celestiaorg/celestia-app/issues/4507